psga/config/psga-schema-sars-cov-2.yaml

```diff
@@ -322,6 +322,10 @@ output_fields:
     field_type: string
     required: true
 
+  - field_name: GENOTYPING
+    field_type: dictionary
+    required: true
+
```

Comment:
I guessed this field_type.

Reply:
Need to check with @Aiky30 if this will break anything

Reply:
We only check for type on the date, it's not used anywhere else. There is a map to handle this in the future: https://github.com/Congenica/psga-backend/blob/5201b2c607ebdcd5514f572482553a06d5d4a6ef/data_management/metadata_validator.py#L42

Reply:
We will need to port this change into the PSGA-Backend, and then bring the config from the backend back into this project. They are very far apart atm where the backend has moved on.

Reply:
Devops are in progress on getting pipelines to install using their actual config. as this is a change to results (which isn't used currently) I think we can wait for that, but good call out.
From the sound of it this should be safe to go with for now though